ddagent.py

```diff
@@ -407,7 +424,8 @@ def _postMetrics(self):
 
     def run(self):
         handlers = [
-            (r"/intake/?", AgentInputHandler),
+            (r"/intake/metrics?", MetricsAgentInputHandler),
+            (r"/intake/metadata?", MetadataAgentInputHandler),
```

Comment:
Mmh, should we use different endpoints like that, our use different base URLs/ELBs ? or both?
cc @remh 

We need to be in sync for that one.